deps.nix

```diff
@@ -0,0 +1,134 @@
+{ fetchNuGet }: [
+  (fetchNuGet { pname = "coverlet.collector"; version = "3.1.2"; sha256 = "0gsk2q93qw7pqxwd4pdyq5364wz0lvldcqqnf4amz13jaq86idmz"; })
```

Comment:
Interesting, is there way to simplify this? It may become problematic to maintain whole list for large applications. Or that's whole point? Maybe there a way to group set of nugets together?